content/3.Joining Corda Network/4.Onboarding Workflow.md

```diff
@@ -33,29 +39,35 @@ i. The node operator fills out fields in bold in the table below, for every node
+- "L" and "C" City and Country should be the same as that of your entity's official registered address.
+- "OU" Organisation Unit is a discretionary field for you to put in a value of your choice
+- Fields in non-bold are optional, except for State (S), which is required only for legal entities registered in a country with federal states system (e.g. USA, Australia, Canada, etc).
+- The X.500 name is checked for uniqueness. If a CSR is submitted (or a certificate exists) with an X.500 name already in Corda Network, it will be automatically rejected.
 
-iv. The network operator will give the all clear to node operator to submit their CSR.
+2. What is the official registered address of the legal entity?
+3. Are you submitting the node on behalf of your entity, or your sponsee's entity?
+4. Is the legal entity is on a sanction list observed by Corda Network?
 
-### Before you begin...
+- You can refer to Section 1.2 of the [Admission Criteria](https://corda.network/corda-network-rulebook/admissions-criteria/) for the list of sanction lists observed by Corda Network.
 
-- Fields in non-bold are optional, except for State (S), which is required only for legal entities registered in a country with federal states system (e.g. USA, Australia, Canada, etc).
+iii. The Corda Network oeprator (R3 currently) will perform due diligence check on the proposed identity name you submitted. This includes:
 
-- The email address used below should be an actively monitored mailbox. We will use this email address for all communications with the node, including those involved in identity verification. We recommend this is an ‘admin@’ or ‘info@’ generic email associated with the legal entity, to avoid key man risk.
+1. **Legal entity check** - we will check if the legal entity (Organisation field, or "O" field) of the registering node is an officially registered company. *Note: We do not accept individuals joining the Corda Network at the moment.*
+2. **Sanction check** - we will check if the legal entity is on a sanction list observed by the Corda Network operator
+3. **Naming convention check** - we will check if your proposed x500 name follows the all naming convention rules set out in the [Corda Network Rulebook](https://corda.network/corda-network-rulebook/introduction).
 
-- The combination of O and OU are checked for uniqueness. If a CSR is submitted (or a certificate exists) with these two fields already in Corda Network, it will be rejected.
+### Other things to note...
 
-- If you are unsure of how to populate any fields, please see the guidance at the end of the email or reply to this email with your specific questions.
+- If you are setting up a notary, there are additional considerations you need to keep in mind, please check [Customer Hub](https://customerhub.r3.com/) for instructions to set up a notary.
```

Comment:
afaik the notary set up instructions are not there. The only public-facing document is this> https://docs.corda.net/docs/corda-enterprise/4.8/corda-network/setup-segregated-notary.html#setting-up-a-notary-in-a-segregated-network